fotoapparat/src/main/java/io/fotoapparat/FotoapparatBuilder.java

```diff
@@ -40,20 +43,21 @@
     CameraProvider cameraProvider = v1();
     CameraRenderer renderer;
 
-    SelectorFunction<LensPosition> lensPositionSelector = firstAvailable(
+    SelectorFunction<Collection<LensPosition>, LensPosition> lensPositionSelector = firstAvailable(
             back(),
             front(),
             external()
     );
-    SelectorFunction<Size> photoSizeSelector = biggestSize();
-    SelectorFunction<Size> previewSizeSelector = biggestSize();
-    SelectorFunction<FocusMode> focusModeSelector = firstAvailable(
+    SelectorFunction<Collection<Size>, Size> photoSizeSelector = biggestSize();
+    SelectorFunction<Collection<Size>, Size> previewSizeSelector = biggestSize();
+    SelectorFunction<Collection<FocusMode>, FocusMode> focusModeSelector = firstAvailable(
             continuousFocus(),
             autoFocus(),
             fixed()
     );
-    SelectorFunction<Flash> flashSelector = FlashSelectors.off();
-    SelectorFunction<Range<Integer>> previewFpsRangeSelector = Selectors.nothing();
+    SelectorFunction<Collection<Flash>, Flash> flashSelector = FlashSelectors.off();
+    SelectorFunction<Collection<Range<Integer>>, Range<Integer>> previewFpsRangeSelector = Selectors.nothing();
```

Comment:
If it is not a lot of effort for you, with new version of `SelectorFunction` it should be `SelectorFunction<Range<Integer>, Integer>` (same way as it is with `sensorSensitivitySelector`). Meaning is - we select one FPS from given range (not selecting one range from given collection of ranges).

Reply:
Almost. Since we're using `setPreviewFpsRange(min, max)`, we should provide 2 values. Furthermore, we're losing flexibility in Fotoapparat API.

Reply:
Ah, right. Makes sense. Let's keep it as it is.